ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,48 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+RSM:
+  guid: "e6aa94d5-1860-4a5d-8269-0edb968895e1"
+  description: "Run status monitoring"
+  is_abstract: true
+  uses:
+  - run_status
+  opt_uses:
+  - failed_alarm
+  implements:
+  - OPERATIONAL
+
+SEFC:
+  description: "Supply fan and exhaust fan control."
+  is_abstract: true
+  uses:
+  - supply_fan_run_command
+  - supply_fan_speed_percentage_command
+  - exhaust_fan_run_command
+  - exhaust_fan_speed_percentage_command
+  - supply_air_flowrate_setpoint
+  - exhaust_air_flowrate_setpoint
+  opt_uses:
+  - failed_supply_fan_alarm
+  - failed_exhaust_fan_alarm
+  - return_fan_speed_percentage_command
+  implements:
+  - CONTROL
+
+FAM:
+  description: "Fire Alarm monitoring"
+  is_abstract: true
+  uses:
+  - fire_alarm
+  implements:
+  - MONITORING
+
+HRC:
+  description: "Heat recovery run mode control"
+  is_abstract: true
+  uses:
+  - heat_recovery_run_command
+  - heat_recovery_run_mode
```

Comment:
Why do we have the mode and the command? Doesnt the mode indicate what the command is doing?